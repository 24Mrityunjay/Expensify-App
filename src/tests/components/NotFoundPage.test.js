import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import NotFoundPage from '../../components/NotFoundPage';

const expenses = [{
    id: '1',
    description: 'Gum',
    note: '', 
    amount: 195,
    createdAt: 20
}, {
    id: '2',
    description: 'Rent',
    note: '', 
    amount: 109500,
    createdAt: 1000
}, {
    id: '3',
    description: 'Credit Card',
    note: '', 
    amount: 4500,
    createdAt: -1000
}];

test("Should test Expense Dashboard Page", () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(<NotFoundPage />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});