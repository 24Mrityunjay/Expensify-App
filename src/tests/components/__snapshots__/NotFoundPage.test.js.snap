// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Expense Dashboard Page 1`] = `
<div>
  <h2>
    404! Error Not Found
  </h2>
</div>
`;
